README.md

```diff
@@ -1,2 +1,36 @@
+![Github CI/CD](https://github.com/rsksmart/pmt-builder/actions/workflows/workflow.yml/badge.svg)
+
 # pmt-builder
-PartialMerkleTree builder lib
+
+PartialMerkleTree builder library.
+
+This library can be used to build a partial merkle tree for a block of transactions. A partial merkle tree is a pruned version of a full merkle tree, containing only the parts needed to prove a given transaction is part of the tree. 
+
+## Installation
+
+> npm install @rsksmart/pmt-builder
+
+### Install dependencies
+
+> npm install
```

Comment:
What's the difference between this and the above?

Reply:
This Installs The node_modules